### ISSUE TITLE
SyntaxError: Unexpected token function

### ISSUE THREAD

**Issue by fhamborg** [santinic/how2#98]
latest version, fresh install.

```
(base) ➜  ~ how2 python sort
/Users/felix/.npm-global/lib/node_modules/how-2/lib/how2.js:22
async function fetchQuestionAnswers(parsedLink) {
      ^^^^^^^^
SyntaxError: Unexpected token function
    at Object.exports.runInThisContext (vm.js:76:16)
    at Module._compile (module.js:542:28)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/felix/.npm-global/lib/node_modules/how-2/lib/index.js:7:12)
    at Module._compile (module.js:570:32)
(base) ➜  ~
```

**Comment by PaoloCifariello** [comment]
Hi @fhamborg, what version of node are you using to run how2?

**Comment by fhamborg** [comment]
It's `v6.9.5`

**Comment by PaoloCifariello** [comment]
how2 uses `async/awat`, which are available from version `7.6` of node

**Comment by fhamborg** [comment]
Oh, I'm sorry! Upgraded to 10.16 and it's working fine! Thank you! :-)
